src/main/resources/com/uber/jenkins/phabricator/HarbormasterConduitNotifier/config.jelly

```diff
@@ -0,0 +1,6 @@
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+  <f:entry title="Build Target" field="buildTarget"
+           description="Name of the env variable containering the Build Target PHID">
```

Comment:
can we just use PHID and the constant defined elsewhere, or is there a good reason why people would need to populate this separately?